<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditListDialog</class>
 <widget class="QDialog" name="EditListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditListDialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>331</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>21</y>
     <width>26</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Edit item</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="allowEditsButton">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>21</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Allow editing</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/pencil.png</normaloff>:/images/pencil.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>21</y>
     <width>26</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete item</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/trash.png</normaloff>:/images/trash.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>21</y>
     <width>26</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add new item</string>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/newitem.png</normaloff>:/images/newitem.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>21</y>
     <width>16</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
